game/view/buffer.lua

```diff
@@ -115,9 +120,39 @@ function BufferView:draw()
   local text = self.format:format(self.amount)
   local limit = self.sprite:getWidth() * _SCALE + self.font:getWidth(text)
               + _MX/3
-  self.font:set()
   g.push()
   g.translate(self.pos.x, self.pos.y)
+
+  --Draw button ontop of front buffer
+  if self.side == "front" then
+     local scale = .4
+     local b_y = -115
+     local b_x = -5
+     g.setColor(1,1,1)
+     g.draw(self.button, b_x, b_y, nil, scale)
+     --Draw "draw hand" text
+     local text = "draw hand"
+     local gap = 10
+     local text_y = b_y - 5
+     local text_x = b_x + self.button:getWidth()*scale + gap
+     self.text_font:set()
+     g.setColor(COLORS.BLACK)
+     g.print(text, text_x + 2, text_y + 2)
+     g.setColor(COLORS.NEUTRAL)
+     g.print(text, text_x, text_y)
+     --Draw cost of consumption
+     text_y = text_y + 22
+     self.text_font2:set()
+     local cost = self.route:getControlledActor():getBody():getConsumption()
+     local text = "-"..cost.." PP"
+     g.setColor(COLORS.BLACK)
+     g.print(text, text_x + 1, text_y + 1)
+     g.setColor(COLORS.PP)
+     g.print(text, text_x, text_y)
+  end
+
```

Comment:
is there any way this could be in a separate method? for readability

Reply:
I want to create a separate class for face buttons that appear on screen, to solve ssue #949. I'll remove those lines of code then, but for this PR I'm thinking of leaving it.